### DISCUSSION TITLE
Loading big pandas

### DISCUSSION THREAD

**Discussion by 00houssam00** [kernc/backtesting.py#1121] [General]
Hello,

Thank you for this amazing library.

I would like to ask whats the best way to handle large data? is there a way to process the data in chunks and load one chunk at a time into Backtesting (ex:- feeding the backtest.run() with prices one chunk at a time)?

**Comment by gsnoopy** [comment]
You can use chunksize parameter to divide your data in blocks (chunk) and the size is determined by you

```
for chunk in pd.read_csv(filepath, chunksize=10000)
    # Process each chunk
    backtest.run(chunk)
```
